doc/xml/user-guide.xml

```diff
@@ -740,6 +740,154 @@
                 </execute>
             </execute-list>
         </section>
+
+        <!-- SECTION => RESTORE - DELTA -->
+        <section id="option-db-include">
+            <title>Restore Selected Databases</title>
+
+            <p>There may be cases where it is desirable to selectively restore specific databases from a cluster backup.  This could be done for performance reasons or to move selected databases to a machine that does not have enough space to restore the entire cluster backup.</p>
+
+            <p>To demonstrate this feature two databases are be created: test1 and test2.  A new backup is run so <backrest/> is aware of the new databases.</p>
```

Comment:
typo: To demonstrate this feature two databases are TO be created (or just say "are created")